app/services/application_state_change.rb

```diff
@@ -5,11 +5,11 @@ class ApplicationStateChange
   # as they do not need references or the 7-day cooling off period
   STATES_THAT_MAY_BE_SENT_TO_PROVIDER = %i[application_complete unsubmitted].freeze
   STATES_NOT_VISIBLE_TO_PROVIDER = %i[unsubmitted awaiting_references application_complete cancelled].freeze
-  STATES_VISIBLE_TO_PROVIDER = %i[awaiting_provider_decision offer pending_conditions recruited rejected declined withdrawn conditions_not_met offer_withdrawn rejected_at_end_of_cycle].freeze
+  STATES_VISIBLE_TO_PROVIDER = %i[awaiting_provider_decision offer pending_conditions recruited rejected declined withdrawn conditions_not_met offer_withdrawn application_not_sent].freeze
```

Comment:
I think this is wrong? The application is never sent to the provider, so it should not be visible to providers?

Reply:
There was some discussion with Duncan and Theo about whether this should be exposed. They decided it should be. Happy to discuss with them 👍

Reply:
Pinging @duncanjbrown @tvararu. Feels like we shouldn't expose these to providers?

Reply:
Looking back at this I agree it should be treated the same as unsubmitted and therefore should not be visible.